collections/guides/manage/manage-reactor-formulas-terraform.md

```diff
@@ -0,0 +1,174 @@
+---
+layout: default
+title: Manage Reactor Formulas with Terraform
+permalink: /guides/manage-reactor-formulas-with-terraform/
+categories: guides
+subcategory: manage
+nav_order: 2
+has_children: true
+has_toc: false
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+
+# Manage Reactor Formulas with Terraform
+{: .no_toc }
+
+Terraform is a popular and powerful tool that enables you to version your infrastructure as code. In this guide, we'll walk
+you through setting up Terraform to manage your resources locally within Basis Theory. By the end of this guide, you will have
+created a Reactor Formula under your Tenant using Terraform.
+
+### Table of contents
+{: .no_toc .text-delta }
+1. 
+{:toc}
+
+<span class="base-alert warning">
+  <span>
+    To start, you'll need a new <code>Management</code> [Application](https://docs.basistheory.com/api-reference/#applications) with <code>reactor:create</code>, <code>reactor:read</code>, <code>reactor:update</code>, and <code>reactor:delete</code> permissions. <a href="https://portal.basistheory.com/applications/create?name=Terraform+Application&type=management&permissions=reactor%3Acreate&permissions=reactor%3Aread&permissions=reactor%3Aupdate&permissions=reactor%3Adelete" target="_blank">Click here to create one.</a>
+  </span>
+</span>
+
+<span class="base-alert warning">
+  <span>
+    Creating custom Reactor Formulas is currently in Private Beta. If you would like to be added to the beta program, please <a href="mailto:support@basistheory.com?subject=Reactor Beta Access">contact us</a>!
```

Comment:
I've been using this link to create a support ticket instead of an email:
https://support.basistheory.com/hc/requests/new?tf_subject=Join%20Beta%20Program%20to%20Code%20Your%20Own%20Reactor&amp;tf_description=Let%20us%20know%20what%20you'd%20like%20to%20do%20with%20your%20Reactor&amp;tf_priority=normal